dcos-oauth/locallogin.go

```diff
@@ -18,7 +19,6 @@ import (
 const (
 	localLoginEnabled = true  //TODO: This should come from a config file somewhere
```

Comment:
This will be read from systemd config file where other settings come from in the near future. Ethos issue #2425.